src/main/java/com/google/sps/EventMarker.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.EventDateTime;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.time.*;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date; 
+
+/**
+ * Marker data for each event to be added to the map.
+ */
+ public class EventMarker {
+    private String name;
+    private String description;
+    private String location; 
+    DateTime date; 
+    String dateOutput; 
+    private String groupName;
+    private int year; 
+    private int month; 
+    private int day; 
+
+    public EventMarker(String name, String description, String location, DateTime date, String groupName) {
+        this.name = name;
+        this.description = description;
+        this.location = location;
+        this.date = date;
+        this.groupName = groupName;
+        formatDate();
+    } 
+
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Formats the Date ex. Monday, June 08, 2000 12:30PM
+     */
+    private void formatDate() {
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("EEEE, MMMM dd, yyyy hh:mm a");
```

Comment:
This can be a static final variable for the class since it doesn't change between function calls.